# DetectedPerson.msg

sensor_msgs/CompressedImage image
sensor_msgs/RegionOfInterest bbox